libraries/ESPmDNS/src/ESPmDNS.cpp

```diff
@@ -59,6 +59,7 @@ bool MDNSResponder::begin(const char* hostName){
     }
     WiFi.onEvent(_on_sys_event);
     _hostname = hostName;
+	_hostname.toLowerCase();
```

Comment:
shouldn't this be merged with the line above as:
`_hostname = hostName.toLowerCase();`

Reply:
I don't think
String.toLowerCase() make side effect on the self object and doesn't returns
void toLowerCase(void);

https://www.arduino.cc/reference/en/language/variables/data-types/string/functions/tolowercase/

Reply:
You are right, I was looking at it and it looked odd. Normally a method like this would return the modification rather than do the modification on the object itself. It appears that https://github.com/espressif/arduino-esp32/blob/master/cores/esp32/WString.h#L255 is just doing a direct modification on the object in place.